TrackletAlgorithm/TrackletEngine.h

```diff
@@ -38,13 +38,12 @@ enum phireg {
 //----------------------------
 // Tracklet Engine main code
 //============================
-template<int innertype, int outertype, unsigned int ptdepth,
+template<int innertype, int outertype,
   unsigned int stubptinnerdepth, unsigned int stubptouterdepth>
 void TrackletEngine(
 		    const ap_uint<3> bx,
 		    const VMStubTEInnerMemory<innertype>& instubinnerdata,
 		    const VMStubTEOuterMemory<outertype>& instubouterdata,
-                    const ap_uint<1> pttable[ptdepth],
                     const ap_uint<1> bendinnertable[stubptinnerdepth],
                     const ap_uint<1> bendoutertable[stubptouterdepth],
```

Comment:
are these new magic numbers that should be rooted out? or maybe ap_unit<1> should be a typedef of something here? I guess it's just a single bit LUT, do we do this elsewhere?

Reply:
There are single-bit LUTs in the VMR and ME, and as far as I can tell, nobody has created a typedef or anything for this type.

I tend to think there's not much magic when we're down to a single bit. I'm not necessarily opposed to something like a typedef, but I don't feel like it would add any clarity to the code, personally.